### DISCUSSION TITLE
Iconscout 0 bytes download

### DISCUSSION THREAD

**Discussion by champolot** [Kyutefox/Iconify#31] [Q&A]
Currently encountering a problem when downloading a file in iconscout. It only gives me 0 byte file size when downloading a file

**Comment by MrJukeman** [comment]
@champolot New update is coming soon with some new features and bug fixes.

**Reply by champolot** [reply]
Thanks for the update bro! I'm so excited for the new featres. I found out that the bug was from their end. Eventually the bug was fixed the next day. Cheers!
